### PR TITLE
Update object for coupon write test

### PR DESCRIPTION
**Pull request by JeroenSormani**
Please note!
Untested PR, changes made in browser.

Reference;
https://woocommerce.github.io/woocommerce-rest-api-docs/wp-api-v2.html#create-a-coupon

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
-          customer_emails: [],
-          description: ''
-        }
+        code: `conversio-test-${uuid.v1()}`,
+        type: 'percent',
+        amount: '10',
+        individual_use: true,
+        product_ids: [],
+        exclude_product_ids: [],
+        enable_free_shipping: false,
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        free_shipping: false,
```

File: lib/app.js [hunk 2]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
-          maximum_amount: '0.00',
-          customer_emails: [],
-          description: ''
-        }
+        code: `conversio-test-${uuid.v1()}`,
+        type: 'percent',
+        amount: '10',
+        individual_use: true,
+        product_ids: [],
+        exclude_product_ids: [],
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        excluded_product_ids: [],
```

File: lib/app.js [hunk 3]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
-          product_category_ids: [],
-          exclude_product_category_ids: [],
-          exclude_sale_items: true,
-          minimum_amount: '100.00',
-          maximum_amount: '0.00',
-          customer_emails: [],
-          description: ''
-        }
+        code: `conversio-test-${uuid.v1()}`,
+        type: 'percent',
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        discount_type: 'percent',
```

File: lib/app.js [hunk 4]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
+        amount: '10',
+        individual_use: true,
+        product_ids: [],
+        exclude_product_ids: [],
+        enable_free_shipping: false,
+        product_category_ids: [],
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        product_categories: [],
```

File: lib/app.js [hunk 5]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
+        exclude_product_category_ids: [],
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        excluded_product_categories: [],
```

File: lib/app.js [hunk 6]
```diff
@@ -119,22 +119,20 @@ app.route('/return')
...
+        exclude_sale_items: true,
+        minimum_amount: '100.00',
+        maximum_amount: '0.00',
+        customer_emails: [],
```
Comment (jmnsf) [inline lib/app.js]:
```suggestion
        email_restrictions: [],
```

**Review by jmnsf — CHANGES_REQUESTED** [review]
Bunch of keys changed as well :)

**Comment by JeroenSormani** [comment]
Changed the API versions too at some point..?
Are the properties already updated in the app itself, just not the debug?

**Comment by stefanosala** [comment]
thanks @JeroenSormani  :) write test now works

**Pull request merged**